strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/storage/validation/groupId/GroupIdCaseValidatorTest.java

```diff
@@ -0,0 +1,52 @@
+package org.carlspring.strongbox.storage.validation.groupId;
+
+import org.carlspring.strongbox.artifact.coordinates.MavenArtifactCoordinates;
+import org.carlspring.strongbox.providers.io.RepositoryFileAttributes;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Spy;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+
+/**
+ * Created by dinesh on 12/7/17.
+ */
+
+public class GroupIdCaseValidatorTest {
+
+    @Spy GroupIdCaseValidator groupIdCaseValidator = new GroupIdCaseValidator();
+
+    MavenArtifactCoordinates mavenArtifactCoordinates;
+    @Mock
+    RepositoryPath repositoryPath = new RepositoryPath(null,null);
+
+    @Mock
+    RepositoryFileAttributes repositoryFileAttributes ;
+
+
+    @Before
+    public void setUp(){
+        initMocks(this);
+        mavenArtifactCoordinates = new MavenArtifactCoordinates("org.dinesh.artifact.is.Uppercase",
+                "my-maven-artifact",
+                "1.0",
+                "classfier",
+                "extension");
+       // repositoryFileSystem
+    }
+
+
+    @Test(expected = GroupIdValidationException.class)
+    public void validateGroupIdCase() throws Exception {
+       doReturn(repositoryFileAttributes).when(groupIdCaseValidator).getAttributes(any());
+       when(repositoryFileAttributes.getCoordinates()).thenReturn(mavenArtifactCoordinates);
+        groupIdCaseValidator.validateGroupIdCase(repositoryPath);
+    }
+
+}
```

Comment:
Could you please set up you IDE to add new lines to the end of source/resource files?